app/assets/javascripts/actions/projectBoard.js

```diff
@@ -53,3 +68,25 @@ export const fetchProjectBoard = projectId =>
       return dispatch(errorRequestProjectBoard(error));
     }
   };
+
+export const closeSearch = () =>
+  async (dispatch, getState, {}) => {
+    try {
+      dispatch(closeSearchSuccess());
```

Comment:
Is there a case where it may fail?

Reply:
Nope. In this case, can I remove the try/catch?